src/plugins/blocksmith/components/blocksmith/Blocksmith.jsx

```diff
@@ -32,10 +32,8 @@ import createDelegateFactory from '@triniti/app/createDelegateFactory';
 import DraggableTextBlock from '@triniti/cms/plugins/blocksmith/components/draggable-text-block';
 import HighlightButton
   from '@triniti/cms/plugins/blocksmith/components/highlight-inline-toolbar-button';
-import isMacOS from '@triniti/cms/utils/isMacOS';
```

Comment:
@tbone1020 since these utils are no longer used for anything you can remove them entirely

Reply:
Done